src/frequenz/sdk/timeseries/_moving_window.py

```diff
@@ -241,6 +241,47 @@ def capacity(self) -> int:
         """
         return self._buffer.maxlen
 
+    # pylint before 3.0 only accepts names with 3 or more chars
+    def at(self, key: int | datetime) -> float:  # pylint: disable=invalid-name
+        """
+        Return the sample at the given index or timestamp.
+
+        In contrast to the `window` method, which expects a slice as argument,
```

Comment:
Optional, as I will still go through the code docs and improve them, but mentioning this so people get used to use cross references.

```suggestion
        In contrast to the [`window`][frequenz.sdk.timeseries.MovingWindow.window] method, which expects a slice as argument,
```